contracts/FlowTransactionScheduler.cdc

```diff
@@ -75,7 +80,12 @@ access(all) contract FlowTransactionScheduler {
         priority: UInt8,
         executionEffort: UInt64,
         transactionHandlerOwner: Address,
-        transactionHandlerTypeIdentifier: String
+        transactionHandlerTypeIdentifier: String,
+        transactionHandlerUUID: UInt64,
+
+        // The public path of the transaction handler that can be used to resolve views
```

Comment:
couldn't this be retrieved from the Utils contract by the event ID? I feel we should store metadata in the Utils contract and then make the tools retrieve it from there using the ID. This makes the event not contain any non-critical info, and also allows for extensibility down the line, because we now for sure can't predict all the usecases